src/Snppts/Snppts.csproj

```diff
@@ -14,7 +14,6 @@
 
   <ItemGroup>
     <Folder Include="Views\Snippets\" />
-    <Folder Include="Snippets\" />

```

Comment:
I've seen this on multiple PRs now, so it's probably something that happens automatically, but any idea why this is removed?

Reply:
humm.. that's strange...
Visually nothing was changed, but when I edit the "csproj" I can see this remotion

I've tried to download again and made some changes and nothing was removed 🤔
Btw, I've undone these changes

<p align="center">
	<kbd>
		<img src="https://user-images.githubusercontent.com/19656249/68421450-0a166100-017d-11ea-8090-00dc6572020f.png" alt="image" style="max-width:100%;"/>
	</kbd>
</p>

Reply:
> 
> 
> I've seen this on multiple PRs now, so it's probably something that happens automatically, but any idea why this is removed?

I've seen the same thing